logging/api/list_logs.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import time
+
+from gcloud import logging
+from oauth2client.client import GoogleCredentials
+
+LOGGER_NAME = 'mylogger'
+LOG_MESSAGE = 'A simple entry'
+
+
+def write_entry(mylogger, log_message):
+    """Lists all sinks for a project"""
+    print('Writing log entry for logger '.format(mylogger))
+    # [START write]
+    mylogger.log_text(log_message)
+    # [END write]
+
+
+def list_entries(client, logger):
+    """Lists all entries for a logger"""
+    print('Listing all log entries for logger {}'.format(logger.name))
+
+    # [START list]
+    entries = []
+    while True:
+        new_entries, token = client.list_entries(filter_='logName="{}"'.format(
+           logger.full_name))
+        entries += new_entries
+        if token is None:
+            break
+
+    for entry in entries:
+        timestamp = entry.timestamp.isoformat()
+        print('{}Z: {}'.format
```

Comment:
agreed but its gcloud-python, guy who wrote monitoring client has a big thing about how datetimes are handled so will have to review that